systems/notification/mailer/pkg/server/mailer.go

```diff
@@ -22,116 +22,124 @@ import (
 	"github.com/ukama/ukama/systems/notification/mailer/pkg/db"
 )
 
-type EmailData struct {
+type EmailPayload struct {
 	To      []string
-	Subject string
-	Body    string
+	TemplateName string `json:"template_name"`
 	Values  map[string]interface{}
 }
 
+
 type MailerServer struct {
 	mailerRepoRepo db.MailerRepo
```

Comment:
This should be `mailerRepo` instead of `mailerRepoRepo`

Reply:
naming updated